examples/vpc-scenario-3/README.md

```diff
@@ -0,0 +1,50 @@
+# Example to test the VPC Scenario 3 Module
+
```

Comment:
here is where there should be a description of the topology (not a list of resources, but a link to AWS Scenario 3 and a description of how this env compares to that scenario, if and where it is different from the scenario, etc). Include a graphic from AWS if this implements the topology depicted in the graphic. It should be made clear, what traffic is allowed between the subnets, there are implicit assumptions in the current README, but these details need to be explicit and obvious to the reader, with no ambiguity.